src/mainframe/dependencies.py

```diff
@@ -53,5 +53,6 @@ def __call__(self, data: Annotated[AuthenticationData, Depends(validate_token)])
         token_permissions_set = set(token_permissions)  # type: ignore
         required_permissions_set = set(self.required_permissions)
 
-        if not required_permissions_set.issubset(token_permissions_set):  # type: ignore
+        # type: ignore
```

Comment:
The `type: ignore` is on the wrong lime - it should be on the same line as the condition.